streamlit_dashboard/dashboard.py

```diff
@@ -134,11 +134,24 @@ def scrape_from_steam_html(html_content: bytes, url: str) -> dict:
     regular_price_element = s.find("div", class_="game_purchase_price price", attrs={
         "data-price-final": True})
 
+    image_element = s.find("img", class_="game_header_image_full")
+    description_element = s.find("div", class_="game_description_snippet")
```

Comment:
Thank you for adding in the image and description to the dashboard, this will make it a lot more visually appealing!